### ISSUE TITLE
XHalfLap doesnt run outside Rhino?

### ISSUE THREAD

**Issue by chenkasirer** [gramaziokohler/compas_timber#179]

**Comment by chenkasirer** [comment]
could be `sorted` with the `zip` there is behaving differently in CPython vs. IronPython (or just 2.7 vs. 3)

**Comment by obucklin** [comment]
this function only returns the planes anyways, so we can just sort planes with `planes_angles` as key. I think that's how I did it in the other laps. I'll change it real quick

**Comment by obucklin** [comment]
@chenkasirer maybe you can test it real quick. I just changed it

**Comment by chenkasirer** [comment]
yup seems to have worked. I wrote a few unittests you can find them in `tests/compas_timber/`
you run them using `invoke test` and they should all pass.

When opening a PR or when pushing new commits to it, it'll run them automatically as Github Actions.
You can click on `Details` next to an action that failed to checkout why it failed. All checks should be green before we can merge.

![Image](https://github.com/gramaziokohler/compas_timber/assets/3398309/3a7db8a2-354f-4d9e-8143-e5303b2b264c)
